bin/main.rb

```diff
@@ -1,69 +1,75 @@
 #!/usr/bin/env ruby
-# Game Pseudocode
-# 1.Create board
-# 2.Get user input from players
-# 3.Track players and turns
-# 4.Check if board is full
-# 5.check for valid move
-# 6.check positions for each player
-# 7.check for winner, loser and draw
-# 8. display output
-#
+# rubocop:disable Layout/LineLength
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Metrics/MethodLength
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/PerceivedComplexity
+require './lib/player.rb'
+require './lib/board.rb'
```

Comment:
- [ ] Make sure the path to the Board and Player classes is correct, you have to navigate out of the current folder(../)

- [ ] You should use require_relative for files inside your own project.